drivers/aufs/aufs.go

```
@@ -681,7 +681,7 @@ func (a *Driver) Cleanup() error {
 func (a *Driver) aufsMount(ro []string, rw, target string, options graphdriver.MountOpts) (err error) {
 	defer func() {
 		if err != nil {
-			Unmount(target)
+			_ = Unmount(target)
```

Should we log those ignored errors for visibility?